### PR TITLE
[leofs][doc] Implement Prometheus support

### PR DESCRIPTION
**Pull request by yosukehara**
This PR includes each component of `snmp.yml` for Prometheus/SNMP-Exporter, modified each reltool's file, and modified  the documentation of `Admin/System Admin/Monitoring`.

* [Admin/System Admin/Monitoring](https://yosukehara.github.io/leofs/admin/system_admin/monitoring/#using-prometheus-grafana)

### CODE REVIEW

File: docs/admin/system_admin/monitoring.md [hunk 1]
```diff
@@ -258,16 +258,40 @@ SNMPv2-SMI::enterprises.35450.34.34.0 = Gauge32: 88735915
...
 
+## Using Prometheus + Grafana
+
+We provide another way of LeoFS system monitoring which uses Prometheus [^4] and SNMP-Exporter [^5], and deliver `snmp.yml` of each component *(LeoGateway, LeoManager, and LeoStorage)* so that you can easily start.
+
+* <a href="https://github.com/leo-project/leofs/blob/v1/apps/leo_gateway/priv/leo_gateway_snmp.yml" target="_blank">LeoGateway snmp.yml</a>
+* <a href="https://github.com/leo-project/leofs/blob/v1/apps/leo_manager/priv/leo_manager_snmp.yml" target="_blank">LeoManager snmp.yml</a>
+* <a href="https://github.com/leo-project/leofs/blob/v1/apps/leo_storage/priv/leo_storage_snmp.yml" target="_blank">LeoStorage snmp.yml</a>
+
+If you get a LeoFS' package, it includes each `snmp.yml` under each `etc` directory as below:
```
Comment (mocchira) [inline docs/admin/system_admin/monitoring.md]: "If you get a LeoFS' package" can be rephrased  as "Within a LeoFS' package"

File: rel/leo_manager/reltool.config.in [hunk 2]
```diff
@@ -130,5 +130,7 @@
            {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/context.conf",   "snmp/snmpa_manager_1/agent/conf/context.conf"},
            {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/standard.conf",  "snmp/snmpa_manager_1/agent/conf/standard.conf"},
            {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/vacm.conf",      "snmp/snmpa_manager_1/agent/conf/vacm.conf"},
-           {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/notify.conf",      "snmp/snmpa_manager_1/agent/conf/notify.conf"}
+           {copy, "../../apps/leo_manager/snmp/snmpa_manager_1/agent/conf/notify.conf",      "snmp/snmpa_manager_1/agent/conf/notify.conf"},
+           %% snmp-related
```
Comment (mocchira) [inline rel/leo_manager/reltool.config.in]: "snmp-related" should be "prometheus-related" as well as other reltool.config.in.

**Review by mocchira — CHANGES_REQUESTED** [review]
Almost all LGTM. Two nitpicks there so please check those out.

**Comment by yosukehara** [comment]
@mocchira PTAL

**Review by mocchira — APPROVED** [review]

**Comment by mocchira** [comment]
LGTM.

**Pull request merged**